pkgs/development/tools/reindeer/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, libiconv
+, darwin
+, stdenv
+, nix-update-script
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "reindeer";
+  version = "unstable-2023-05-18";
+
+  src = fetchFromGitHub {
+    owner = "facebookincubator";
+    repo = pname;
+    rev = "2d6d37dcddaa840b6cd99951176c972f4c3afddd";
+    sha256 = "sha256-31Kf6cVqAVFprTHEJp06E0p0y5YwdowhmPti8JBJw3g=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  postPatch = ''
+    ln -s ${./Cargo.lock} Cargo.lock
+  '';
+
+  doCheck = false;
```

Comment:
please add a comment explaining why

Reply:
This was carryover from the buck2 PR (whose tests were not always passing), so I just tested locally and I think we can remove this!